files.json

```diff
@@ -467,6 +467,14 @@
     "SHA": "2d75848ac606061efe52a8068d0e647b35ce487a15bb52272c427df485193602",
     "URL": "https://storage.googleapis.com/golang/go1.15.linux-amd64.tar.gz"
   },
+  "go1.16.linux-amd64.tar.gz": {
```

Comment:
Please remove this, files should only be listed here as they appear at https://golang.org/dl/ or similar. Using 1.16beta1 when `go1.16` is requested is controlled by data.json.